templates/web/src/sdk.ts.twig

```diff
@@ -319,12 +319,15 @@ class {{ spec.title | caseUcfirst }} {
             const cookieFallback = response.headers.get('X-Fallback-Cookies');
 
             if (typeof window !== 'undefined' && window.localStorage && cookieFallback) {
-                window.console.warn('Appwrite is using localStorage for session management. Increase your security by adding a custom domain as your API endpoint.');
+                window.console.warn('{{spec.title | caseUcfirst}} is using localStorage for session management. Increase your security by adding a custom domain as your API endpoint.');
                 window.localStorage.setItem('cookieFallback', cookieFallback);
             }
 
             return data;
         } catch (e) {
+            if (e instanceof {{spec.title | caseUcfirst}}Exception) {
+                throw e;
+            }
```

Comment:
What exactly caused the missing error code?

Reply:
It was a nested exception caught and then thrown by the parent try/catch.